pkg/logs/input/file/file_provider.go

```diff
@@ -55,7 +55,7 @@ func (p *Provider) FilesToTail() []*File {
 	shouldLogErrors := p.shouldLogErrors
 	p.shouldLogErrors = false // Let's log errors on first run only
 
-	sources := p.sources.GetValidSourcesWithType(config.FileType)
+	sources := p.sources.GetSourcesWithType(config.FileType)
```

Comment:
I think we don't need `GetValidSourcesWithType` anymore and replace all calls by `GetSourcesWithType `

Reply:
Hmm we still want to filter out sources that stem from bad configurations in other places I think, I will check